processor/elasticinframetricsprocessor/config.go

```diff
@@ -20,6 +20,7 @@ package elasticinframetricsprocessor // import "github.com/elastic/opentelemetry
 type Config struct {
 	AddSystemMetrics bool `mapstructure:"add_system_metrics"`
 	AddK8sMetrics    bool `mapstructure:"add_k8s_metrics"`
+	Override         bool `mapstructure:"override"`
```

Comment:
Is this setting going to be documented anywhere?

Reply:
Updated Readme https://github.com/elastic/opentelemetry-collector-components/pull/147/files#diff-57ed28cac77217dd4bd0ef0633c9dd5d9294a2760cb3f7ecb22a59c199d750da